cmd/common/common.go

```diff
@@ -1,15 +1,15 @@
-package client
```

Comment:
Keep this file in client, because crypto/keys/client can be used to craft custom client without relying on the rest of our cmds.

That’s what I’m doing for the blog client on https://github.com/gnolang/blog.

And, cmd/* should always be `package main`.

Reply:
Got it, I've moved out the `gnokey` subcommand structure back to the `client` package where it was initially, and just created a link to it in the `cmd/gnokey` directory:

[9df7403](https://github.com/gnolang/gno/pull/497/commits/9df7403574d78fc11ec11b4a7ed4d53e632c56a0)